src/main/java/org/fairdatapoint/config/BootstrapConfig.java

```diff
@@ -63,30 +62,28 @@
 @Slf4j
 public class BootstrapConfig {
     private final ResourcePatternResolver resourceResolver = new PathMatchingResourcePatternResolver();
+    private final BootstrapProperties bootstrap;
     private final FixtureHistoryRepository fixtureHistoryRepository;
-    private final boolean bootstrapEnabled;
-    private final Path dbFixturesPath;
     private final List<Resource> resources = new ArrayList<>();
 
-    public BootstrapConfig(
-            FixtureHistoryRepository fixtureHistoryRepository,
-            @Value("${bootstrap.enabled:false}") boolean bootstrapEnabled,
-            @Value("${bootstrap.db-fixtures-dir}") String dbFixturesDir
-    ) {
-        this.bootstrapEnabled = bootstrapEnabled;
-        this.dbFixturesPath = Path.of(dbFixturesDir);
+    public BootstrapConfig(BootstrapProperties bootstrapProperties, FixtureHistoryRepository fixtureHistoryRepository) {
+        this.bootstrap = bootstrapProperties;
         this.fixtureHistoryRepository = fixtureHistoryRepository;
     }
 
     @Bean
     public Jackson2RepositoryPopulatorFactoryBean repositoryPopulator() {
         final Jackson2RepositoryPopulatorFactoryBean factory = new Jackson2RepositoryPopulatorFactoryBean();
-        if (bootstrapEnabled) {
+        if (this.bootstrap.isEnabled()) {
             log.info("Bootstrap repository populator enabled");
             try {
                 // collect fixture resources
-                final Path fixturesPath = dbFixturesPath.resolve("*.json");
-                resources.addAll(List.of(resourceResolver.getResources("file:" + fixturesPath)));
+                log.info("Looking for db fixtures in the following directories: {}",
+                        String.join(", ", this.bootstrap.getDbFixturesDirs()));
+                for (String fixturesDir : this.bootstrap.getDbFixturesDirs()) {
+                    final String locationPattern = "file:" + Path.of(fixturesDir).resolve("*.json");
```

Comment:
The `Path.of(fixturesDir).resolve("*.json")` is probably causing the failure on Windows...